jobs/upload-kibana-objects/templates/bin/run

```diff
@@ -103,11 +100,11 @@ def process_bulk_query_result(result)
 end
 
 log_status("======= Defaults to upload:")
-upload_data = render_upsert_queries('index-pattern')
-upload_data += render_upsert_queries('config')
-upload_data += render_upsert_queries('search')
-upload_data += render_upsert_queries('visualization')
-upload_data += render_upsert_queries('dashboard')
+
+upload_data = ""
+p("kibana_objects.upload_patterns").each do |pattern|
+  upload_data += render_upsert_queries(pattern["type"], pattern["pattern"])
+end
```

Comment:
@jmcarp you forget the delimiters, also I prefer to use fetch method in this case, it will raise the exception if no keys found.
```
<% p("kibana_objects.upload_patterns").each do |pattern| %>
  upload_data += render_upsert_queries("<%= pattern.fetch("type") %>", "<%= pattern.fetch("pattern") %>")
<% end %>
```